packages/terra-tag/src/TagList.jsx

```diff
@@ -0,0 +1,348 @@
+import React, {
+  useCallback, useLayoutEffect, useRef, useState,
+} from 'react';
+import {
+  KEY_LEFT,
+  KEY_RIGHT,
+  KEY_HOME,
+  KEY_END,
+} from 'keycode-js';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import PropTypes from 'prop-types';
+import ResizeObserver from 'resize-observer-polyfill';
+import ThemeContext from 'terra-theme-context';
+import { injectIntl } from 'react-intl';
+import { v4 as uuidv4 } from 'uuid';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+import TagsUtils from './TagsUtils';
+import RollUpTag from './RollupTag';
+import styles from './Tag.module.scss';
+import SharedUtil from './_SharedUtil';
```

Comment:
For organization & readability
```suggestion
import React, { useCallback, useLayoutEffect, useRef, useState } from 'react';
import { KEY_LEFT, KEY_RIGHT, KEY_HOME, KEY_END } from 'keycode-js';

import { injectIntl } from 'react-intl';
import classNames from 'classnames';
import classNamesBind from 'classnames/bind';
import PropTypes from 'prop-types';
import ResizeObserver from 'resize-observer-polyfill';

import { v4 as uuidv4 } from 'uuid';

import VisuallyHiddenText from 'terra-visually-hidden-text';
import ThemeContext from 'terra-theme-context';

import TagsUtils from './TagsUtils';
import RollUpTag from './RollupTag';
import SharedUtil from './_SharedUtil';
import styles from './Tag.module.scss';
```